mappings/net/minecraft/nbt/NbtElement.mapping

```diff
@@ -0,0 +1,37 @@
+CLASS net/minecraft/class_2520 net/minecraft/nbt/NbtElement
+	COMMENT Represents an NBT element.
+	COMMENT <p>
+	COMMENT NBT elements are identified by a type, here's a list of the possible type values:
+	COMMENT <ul>
+	COMMENT <li>{@code 0} - null</li>
+	COMMENT <li>{@code 1} - byte</li>
+	COMMENT <li>{@code 2} - short</li>
+	COMMENT <li>{@code 3} - int</li>
+	COMMENT <li>{@code 4} - long</li>
+	COMMENT <li>{@code 5} - float</li>
+	COMMENT <li>{@code 6} - double</li>
+	COMMENT <li>{@code 7} - byte array</li>
+	COMMENT <li>{@code 8} - string</li>
+	COMMENT <li>{@code 9} - list</li>
+	COMMENT <li>{@code 10} - compound object</li>
+	COMMENT <li>{@code 11} - int array</li>
+	COMMENT <li>{@code 12} - long array</li>
+	COMMENT <li>{@code 99} - number</li>
```

Comment:
I think it would be beneficial to document exactly what the generic number type means here as well.